corehq/apps/app_manager/models.py

```diff
@@ -1042,19 +1047,19 @@ def active_actions(self):
             if self.requires == 'none':
                 action_types = (
                     'open_case', 'update_case', 'close_case', 'subcases',
-                    'usercase_update', 'usercase_preload',
+                    'usercase_update', 'usercase_preload', 'usercase_subcases',
                 )
             elif self.requires == 'case':
                 action_types = (
                     'update_case', 'close_case', 'case_preload', 'subcases',
-                    'usercase_update', 'usercase_preload',
+                    'usercase_update', 'usercase_preload', 'usercase_subcases',
```

Comment:
Looks like you're building on a precedent here, so maybe it's too late to be bringing it up, but all of the names in this list are verbs except for the ones ending with `subcases`. The word "subcases" does not seem like a type of action to me.